BPR.Mediator/Services/CodebaseService.cs

```diff
@@ -20,13 +20,16 @@ private static string ExtractArchive(ArchiveFile file)
     {
         var guid = Guid.NewGuid();
         var directory = Directory.CreateDirectory($"../temp/{guid}");
```

Comment:
Does this actually increase performance?

Reply:
I assumed it wouldn't but wasn't sure if there was some weird shit going on behind the scenes that would make it be the case. But no, it doesn't seem like it